### PR TITLE
SEC-161 - Merge AssetBackedSPVs into other larger ontologies

### PR DESCRIPTION
**Pull request by ElisaKendall**
Signed-off-by: Elisa Kendall <ekendall@thematix.com>

## Description

In this resolution we (1) Replaced DebtSPV with SpecialPurposeVehicle, and the boolean isREMIC with a class describing the concept of a real estate mortgage investment conduit (REMIC) in mortgage-backed securities, allowing the largely redundant AssetBackedSPVs ontology to be eliminated, (2) eliminated a number of circular dependencies among asset-backed security ontologies as well as a few redundant properties in anticipation of further work, (3) merged the contents of the tiny CBOs ontology into the CDOs ontology and eliminated the CBOs ontology, and (4) merged the contents of the tiny CLOs ontology into the CDOs ontology and eliminated the CLOs ontology, thus reducing maintenance overhead and additional circular dependencies.

Fixes: #1568 / SEC-161

## Checklist:

- [x] I'm familiar with the [FIBO developer quide](https://github.com/edmcouncil/fibo/blob/master/CONTRIBUTING.md#contributing-to-the-fibo-code). My contribution meets all the requirements described there.
- [x] My contribution follows the [principles of best practices for FIBO](https://github.com/edmcouncil/fibo/blob/master/ONTOLOGY_GUIDE.md).
- [x] My changes have been reconciled with latest master and no merge conflicts remain.
- [x] This PR is related to exactly one issue. The issue is referenced by using a GitHub keyword such as "fixes", "closes", or "resolves".
- [x] Hygiene tests have been applied by a PR with "(WIP)" in title.
- [x] The issue has been tested locally using a reasoner (for ontology changes).

### CODE REVIEW

File: SEC/Debt/AssetBackedSecurities/MortgageBackedSecurities.rdf [hunk 1]
```diff
@@ -285,22 +300,22 @@
...
+				<owl:onProperty rdf:resource="&fibo-fnd-agr-ctr;definesTermsFor"/>
+				<owl:someValuesFrom rdf:resource="&fibo-sec-dbt-ab-mbs;MortgageDebt"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-sec-dbt-ab-abs;definesTermsFor"/>
-				<owl:someValuesFrom rdf:resource="&fibo-sec-dbt-ab-mbs;MortgageDebt"/>
+				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;isCharacterizedBy"/>
+				<owl:someValuesFrom rdf:resource="&fibo-md-dbtx-aly;MortgageInstrumentWeightedAverageRemainingMaturity"/>
```
Comment (rivettp) [inline SEC/Debt/AssetBackedSecurities/MortgageBackedSecurities.rdf:313]: should be a property not a class

File: SEC/Debt/AssetBackedSecurities/MortgageBackedSecurities.rdf [hunk 2]
```diff
@@ -285,22 +300,22 @@
...
 	
 	<owl:Class rdf:about="&fibo-sec-dbt-ab-mbs;MortgageBackedSecurityInstrument">
-		<rdfs:subClassOf rdf:resource="&fibo-sec-dbt-ab-pbs;PoolBackedSecurityInstrument"/>
+		<rdfs:subClassOf rdf:resource="&fibo-sec-dbt-ab-pbs;PoolBackedSecurity"/>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;isCharacterizedBy"/>
-				<owl:someValuesFrom rdf:resource="&fibo-md-dbtx-aly;MortgageInstrumentWeightedAverageRemainingMaturity"/>
+				<owl:onProperty rdf:resource="&fibo-fnd-agr-ctr;definesTermsFor"/>
+				<owl:someValuesFrom rdf:resource="&fibo-sec-dbt-ab-mbs;MortgageDebt"/>
```
Comment (rivettp) [inline SEC/Debt/AssetBackedSecurities/MortgageBackedSecurities.rdf:307]: surely each pooled mortgage will have its own terms

File: SEC/Debt/AssetBackedSecurities/MortgageBackedSecurities.rdf [hunk 3]
```diff
@@ -502,6 +517,14 @@ Consensus:Review.</skos:editorialNote>
 		<rdfs:label xml:lang="en">private label m b s deal</rdfs:label>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-sec-dbt-ab-mbs;RealEstateMortgageInvestmentConduit">
+		<rdfs:subClassOf rdf:resource="&fibo-sec-fund-fund;SpecialPurposeVehicle"/>
+		<rdfs:label xml:lang="en">real estate mortgage investment conduit</rdfs:label>
+		<skos:definition xml:lang="en">special purpose vehicle or debt instrument that pools mortgage loans together and issues mortgage-backed securities</skos:definition>
+		<fibo-fnd-utl-av:abbreviation xml:lang="en">REMIC</fibo-fnd-utl-av:abbreviation>
+		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">A real estate mortgage investment conduit may be organized as a partnership, a trust, a corporation, or an association and is exempt from federal taxes.</fibo-fnd-utl-av:explanatoryNote>
```
Comment (rivettp) [inline SEC/Debt/AssetBackedSecurities/MortgageBackedSecurities.rdf:525]: make clear that this is the default type in US

File: SEC/Debt/AssetBackedSecurities/MortgageBackedSecurities.rdf [hunk 4]
```diff
@@ -502,6 +517,14 @@ Consensus:Review.</skos:editorialNote>
 		<rdfs:label xml:lang="en">private label m b s deal</rdfs:label>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-sec-dbt-ab-mbs;RealEstateMortgageInvestmentConduit">
+		<rdfs:subClassOf rdf:resource="&fibo-sec-fund-fund;SpecialPurposeVehicle"/>
+		<rdfs:label xml:lang="en">real estate mortgage investment conduit</rdfs:label>
+		<skos:definition xml:lang="en">special purpose vehicle or debt instrument that pools mortgage loans together and issues mortgage-backed securities</skos:definition>
```
Comment (rivettp) [inline SEC/Debt/AssetBackedSecurities/MortgageBackedSecurities.rdf:523]: No, it must be either an instrument or a legal entity (SPV)

File: SEC/Debt/AssetBackedSecurities/PoolBackedSecurities.rdf [hunk 5]
```diff
@@ -132,100 +111,45 @@
...
-		</rdfs:subClassOf>
-		<rdfs:subClassOf>
-			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-sec-dbt-ab-pbs;issuedBy"/>
-				<owl:someValuesFrom rdf:resource="&fibo-sec-dbt-ab-pbs;PoolBackedSecurityIssuer"/>
+				<owl:someValuesFrom rdf:resource="&fibo-sec-sec-pls;DebtPool"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
-		<rdfs:label xml:lang="en">pool backed security instrument</rdfs:label>
+		<rdfs:label xml:lang="en">pool-backed security</rdfs:label>
```
Comment (rivettp) [inline SEC/Debt/AssetBackedSecurities/PoolBackedSecurities.rdf:137]: Pretty unclear

**Review by rivettp — COMMENTED** [review]
Added several comments

File: SEC/Debt/AssetBackedSecurities/AssetBackedSecurities.rdf [hunk 6]
```diff
@@ -89,11 +89,11 @@
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-sec-dbt-ab-abs;ABSIssuer">
-		<rdfs:subClassOf rdf:resource="&fibo-sec-dbt-ab-pbs;PoolBackedSecurityIssuer"/>
+		<rdfs:subClassOf rdf:resource="&fibo-fbc-fi-fi;Issuer"/>
 		<rdfs:subClassOf>
 			<owl:Restriction>
 				<owl:onProperty rdf:resource="&fibo-sec-dbt-ab-abs;describedAs"/>
-				<owl:someValuesFrom rdf:resource="&fibo-sec-dbt-ab-spv;DebtSPV"/>
+				<owl:someValuesFrom rdf:resource="&fibo-sec-fund-fund;SpecialPurposeVehicle"/>
```
[the remark this answers is not in the document]
Comment (ElisaKendall) [reply SEC/Debt/AssetBackedSecurities/AssetBackedSecurities.rdf:96]: @rivettp This property and the ABS Issuer class were eliminated during our FCT discussion

File: SEC/Debt/AssetBackedSecurities/AssetBackedSecurities.rdf [hunk 7]
```diff
@@ -456,21 +447,10 @@
...
-		<rdfs:domain rdf:resource="&fibo-sec-dbt-ab-abs;AutoLoanABS"/>
-		<rdfs:range rdf:resource="&fibo-sec-dbt-ab-cdo;CreditworthinessSelection"/>
-		<skos:definition xml:lang="en">Auto-loans ABS are classified into three categories: prime, non-prime, and sub-prime.</skos:definition>
-	</owl:ObjectProperty>
-	
-	<owl:ObjectProperty rdf:about="&fibo-sec-dbt-ab-abs;definesTermsFor">
-		<rdfs:label>defines terms for</rdfs:label>
-	</owl:ObjectProperty>
-	
 	<owl:ObjectProperty rdf:about="&fibo-sec-dbt-ab-abs;describedAs">
```
[the remark this answers is not in the document]
Comment (ElisaKendall) [reply SEC/Debt/AssetBackedSecurities/AssetBackedSecurities.rdf:450]: @rivettp This property and the ABS Issuer class were eliminated during our FCT discussion

File: SEC/Debt/AssetBackedSecurities/MortgageBackedSecurities.rdf (hunk 3 above)
Comment (ElisaKendall) [reply SEC/Debt/AssetBackedSecurities/MortgageBackedSecurities.rdf:525]: @rivettp We made some changes to this in FCT discussion to clarify.

File: SEC/Debt/AssetBackedSecurities/MortgageBackedSecurities.rdf (hunk 4 above)
Comment (ElisaKendall) [reply SEC/Debt/AssetBackedSecurities/MortgageBackedSecurities.rdf:523]: @rivettp We made this change during our FCT discussion

**Review by rivettp — APPROVED** [review]

**Review by jfgemski — APPROVED** [review]

**Pull request merged**